src/views/forms/AccountManagementForms/SourceSelectMenu.tsx

```diff
@@ -66,11 +70,26 @@ export const SourceSelectMenu = ({
         onSelect(chain.type, 'chain');
       },
       slotBefore: <Styled.Img src={chain.iconUrl} alt="" />,
+      [lowestFeesDecoratorProp]: (
```

Comment:
it seems that we are adding this prop to every item. add it conditionally based on `cctpTokenssByChainId`

Reply:
<img width="457" alt="Screenshot 2024-04-30 at 9 18 17 AM" src="https://github.com/dydxprotocol/v4-web/assets/1062252/4587658e-8ec0-4fd1-a13d-003d6fed260b">

Reply:
🤦 omg lol, my brain